components/tabstrip/tabs-collection.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Tabs Collection
+page_title: Tabs Collection
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs-collection
+tags: telerik,blazor,tab,strip,tabstrip,collection
+published: True
+position: 17
+---
+
+# Tabs Collection
+
+The TabStrip allows you to render its tabs by iterating a collection of configurable objects.
```

Comment:
start with the use case - what is it that you want to do that should land you here. FOr example, something like "declare tabs" or "add tabs" or "manage tabs" or something like that. What does the customer want to do that this article expalins

Reply:
Updated.